docs/src/getting-started/first-steps.md

```diff
@@ -6,14 +6,17 @@ We demonstrate how to create a new project, compile, and test it.
 To start a new project with Starknet Foundry, run `snforge init`
 
 ```shell
-$ snforge init project_name
+snforge init project_name
 ```
 
 Let's check out the project structure
 
 ```shell
-$ cd project_name
-$ tree . -L 1
+cd project_name
+tree . -L 1
+```
+
+```shell
```

Comment:
note: I wonder if having result of `tree` command is better in following form 🤔 :
<img width="500" alt="image" src="https://github.com/user-attachments/assets/61f9afa2-61d7-4d55-945a-abe03133b966">

Imo it should be always visible as the structure of directories is a key point here.